pkg/patterns/addon/pkg/status/conditions.go

```diff
@@ -9,19 +9,34 @@ import (
 )
 
 const (
-	AbnormalReason = "ContainAbnormalTrueConditions"
+	AbnormalReason = "AbnormalTrueConditions"
```

Comment:
It is not clear to me what this actually means. Abnormal-true polarity was a concept in a previous version of the Kubernetes API guidelines that suggested that conditions should be constructed such that a `True` value indicated an abnormal situation. The `Ready` condition that we are using now, is an example of a condition that doesn't follow this concept.

Reply:
How about `HaveAbnormalTrueManifests`? `Ready` condition is for the present object (which does not use the abnormal-true concept), but this `reason` field and its corresponding `message` field are for all the other manifests which this object manages. 

For those manifests, I do applying the abnormal-true polariy: I use kstatus to unify their conditions and pulled out all the abnormal-true conditions to build the value for `message`. So the meaning of `Ready` "false" means the current object has some manifests who are in abnormal-true condition.

Reply:
Eventually I changed this AbnormalReason to `ManifestsNotReady`. 😃